pkg/planner/util/fixcontrol/get.go

```diff
@@ -71,6 +71,9 @@ const (
 	Fix52869 uint64 = 52869
 	// Fix54337 controls whether to apply or not range intersection for index access.
 	Fix54337 uint64 = 54337
+	// Fix58217 controls whether to do HeavyFunctionOptimize. Currently this only applies to Vector data types and
+	// their functions. The HeavyFunctionOptimize eliminate the usage of the function in TopN operators
+	Fix58217 uint64 = 58217
```

Comment:
It should be https://github.com/pingcap/tidb/issues/56318
Expression like `substr` or  `json_extract` can use the same logic.

Reply:
updated.